### PR TITLE
User should be able to add an article under a category

### PR DESCRIPTION
**Pull request by samdiano**
#### What does this PR do?
This pull request creates functionality to add an article under a category and get articles by category

#### Description of Task to be completed?
- Write tests for category model 
- Write the code to add categories to an article and get articles by category

#### How should this be manually tested?
To get all categories: /api/v1/categories -> GET request
To create an article : /api/v1/articles/:articleSlug/comments/:commentId/reply/:replyId/like -> POST request
To get all articles in a category: /api/v1/categoryName/articles -> GET request

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#161250161

### CODE REVIEW

**Review by slimsolz — COMMENTED** [review]
Fix your hound errors

**Review by onwuvic — COMMENTED** [review]
LGTM

File: server/tests/categoryModels.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const invalidCategory = {
+  category: 1
+};
+
+let category;
+let categoryError;
+let stringError;
+
+describe('Category model validations', () => {
+  it('should create categoryt', (done) => {
```
Comment (s-akinrele) [inline server/tests/categoryModels.test.js]: fix this typo

File: server/tests/categoryModels.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        category = newCategory;
+        done();
+      });
+  });
+
+  it('should create category with the correct fields', () => {
+    expect(category.category).to.equal(validCategory.category);
+  });
+
+  describe('category validation (null)', () => {
```
Comment (s-akinrele) [inline server/tests/categoryModels.test.js]: group the validations in a describe block

**Review by s-akinrele — APPROVED** [review]
GTM once the text has been updated.

**Pull request merged**